spec/directConnectionProvider/mockChromeDriver.js

```diff
@@ -0,0 +1 @@
+console.log("mock");
```

Comment:
if this mocks the executable and is actually called, it needs the hashbang on the first line